cli/i18n/I18nFile.php

```diff
@@ -23,7 +26,10 @@ public function load() {
 		return $i18n;
 	}
 
-	public function dump(array $i18n) {
+	/**
+	 * @param array<string,array<array<string>>> $i18n
```

Comment:
I am not sure of the type. It needs to work with:

https://github.com/FreshRSS/FreshRSS/blob/6e7fa07a39e2ff73c3e5e11d38a61d1f2231c859/cli/manipulate.translation.php#L83

Could you please try to do the types for `I18nData`? I believe this is needed to get the types here right

Reply:
ok

Reply:
done